### PR TITLE
Added Colored Manga

### LINKED ISSUE #4124: [Site Request] coloredmanga.com
**Name of the website**
thewebsitename

**Website urls (examples below)**
- Site: https://thewebsiteurl/
- Manga List: https://thewebsiteurl/mangalist.html
- Manga example: https://thewebsiteurl/manga/themanganame.html
- Chapter online viewer example: https://thewebsiteurl/manga/0012/view.html

**Languages**
List of languages supported by the website

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by amit34521**
Closes #4124

### CODE REVIEW

File: src/web/mjs/connectors/ColoredManga.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs'
+
+export default class ColoredManga extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'coloredmanga';
+        super.label = 'Colored Manga';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://coloredmanga.com/';
```
Comment (27136773) [inline src/web/mjs/connectors/ColoredManga.mjs]: May remove trailing `/` slash

File: src/web/mjs/connectors/ColoredManga.mjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs'
```
Comment (27136773) [inline src/web/mjs/connectors/ColoredManga.mjs]: Fix lint error: https://github.com/manga-download/hakuneko/pull/4141/files

File: src/web/mjs/connectors/ColoredManga.mjs (hunk 1 above)
Comment (amit34521) [reply src/web/mjs/connectors/ColoredManga.mjs]: Removed

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**